### PR TITLE
Fixed some formatting

### PR DESCRIPTION
**Pull request by kleinaeinar**
Fixed formatting, added missing links, and fixed the redirect link to this site.

### CODE REVIEW

**Comment by gc** [comment]
we dont need `<br>`, just add 2 spaces for a new line

**Comment by kleinaeinar** [comment]
Should be fixed now :)

File: minions.md [hunk 1]
```diff
@@ -4,64 +4,65 @@ Minions are a feature in Old School Bot that let you simulate playing a virtual
 > Minions are entirely virtual, and not in any way tradeable for real GP or real money. It's simulating the real game for fun. We strictly do not allow any bot users to break any of the official OSRS rules. Read the rules here: [Old School Bot Rules](https://www.oldschool.gg/oldschoolbot/rules)
 
 
-This page is editable by anyone at [obdocs/minions.md]([https://github.com/gc/obdocs/blob/master/minions.md](https://github.com/gc/obdocs/blob/master/minions.md)) - if you notice any mistakes or missing information, you can add it yourself, or report it to us in the [support server](https://discord.gg/ob)
+This page is editable by anyone at [obdocs/minions.md](https://github.com/gc/obdocs) - if you notice any mistakes or missing information, you can add it yourself, or report it to us in the [support server](https://discord.gg/ob)
 
-# Activities
+# Activities <br>
```
Comment (gc) [inline minions.md:9]: still one here

**Comment by kleinaeinar** [comment]
3rd time the charm

**Pull request merged**